ssl-config/src/main/scala/com/avast/sst/ssl/SslContextModule.scala

```diff
@@ -14,11 +14,14 @@ import scala.language.higherKinds
 
 object SslContextModule {
 
+  private[this] val referenceConfig = ConfigFactory.defaultReference().getConfig("ssl-config")
+
   /** Initializes [[javax.net.ssl.SSLContext]] from the provided config. */
-  def make[F[_]: Sync](config: Config): F[SSLContext] = Sync[F].delay {
+  def make[F[_]: Sync](config: Config, withReference: Boolean = true): F[SSLContext] = Sync[F].delay {
```

Comment:
Please provide some ScalaDoc explanation of this parameter.

Reply:
And it would be good to use it in the test we have and maybe example (if it is there, I dont't remember) so that people can see it in action.

Reply:
added tests